R/QCplots.R

```diff
@@ -9,9 +9,9 @@
 #' By default, horizontal reference lines are drawn at the median and +/- n SDs based on the
 #' hlineSD argument. These are statistical reference points, NOT pass/fail limits.
 #'
-#' @param DGEdata A DGEobj object with an an item of type "alignQC" (Required)
+#' @param DGEdata A DGEobj object with an item of type "alignQC" (required)
 #' @param metricNames A list of metrics to plot. Values must exist in column names of the item
-#' of type AlignQC. (Required)
+#' of type AlignQC. (required)
```

Comment:
It is kind of odd to say "required", we usually only point out optional parameters.  Is this consistent throughout the package?

Reply:
ah  I see. I saw it in other functions so I kept it. I can create another PR to remove them all.